robottelo/test.py

```diff
@@ -151,7 +151,10 @@ def setUpClass(cls):  # noqa
     def tearDownClass(cls):
         super(UITestCase, cls).tearDownClass()
         if cls.session_user is not None:
-            cls.session_user.delete(synchronous=False)
+            try:
+                cls.session_user.delete(synchronous=False)
+            except Exception as e:
```

Comment:
The mentioned error is raising requests HTTPError, but as @abalakh mentioned, that part is a "plus" as in fact we do not need to delete every random created user. So if another kind error happens it is not supposed to break the entire automation.  And we can keep an eye at all warning messages logged there.